client/web/src/features/user/userService.ts

```diff
@@ -33,6 +39,19 @@ export class UserService implements ICrudService<IUser> {
       this.api.delete(item.id)
     }
   }
+
+  getInstanceUrls(SeriesInstanceUID: string): Promise<string[]> {
```

Comment:
It is not more logical to put it in seriesService ?

Reply:
oups yes